src/blog/_posts/2018-11-06-opera-57.md

```diff
@@ -0,0 +1,151 @@
+- opera
+license: cc-by-3.0
+---
+
+Opera 57 (based on Chromium 70) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To
+find out what’s new for users, see our
+[Desktop blog post](https://blogs.opera.com/desktop/2018/11/opera-57-stable).
+Here’s what it means for web developers.
+
+## Displaying a dialog will cause pages to leave fullscreen
+
+Dialogs (authentication prompt, payment request et.c) require proper context
+when shown to the user. When the browser is in fullscreen mode - an immersive
+mode - that context is not available. To help the user get the right context
+when a dialog is displayed, the browser will now leave fullscreen mode in those
+cases.
+
+## The `ontouch*` APIs default to disabled on desktop platforms
+
+The `ontouch*` event attributes on `Window`, `Document` and `Element` are
+now disabled by default on desktop platforms. This is to avoid confusion when
+feature detecting touch support - use `navigator.maxTouchPoints` for
+touchscreen detection and `window.TouchEvent` touch event detection instead. To
+support both touch and mouse input, consider using the [Pointer Events](https://w3c.github.io/pointerevents/) API.
+
+Event listeners for the associated touch events can still be added via
+`EventTarget.prototype.addEventListener`, like:
+
+    document.addEventListener("touchstart", ...);
+
+## Options dictionary for postMessage methods
+
+An optional [`PostMessageOptions` dictionary](https://html.spec.whatwg.org/multipage/web-messaging.html#postmessageoptions)
+can now be passed to the
+[`postMessage`](https://html.spec.whatwg.org/multipage/web-messaging.html#dom-window-postmessage-options)
+method on `DedicatedWorkerGlobalScope`, `Worker`, `ServiceWorker`, `ServiceWorker`, and `Window`.
+
+Thus what was previously written as:
+
+    window.postMessage('my message', 'https://origin.example.com');
+
+can now also be written as:
+
+    window.postMessage('my message', { targetOrigin: 'https://origin.example.com' });
+
+## Media and Real-time Communications
+
+* [RTCPeerConnection.prototype.getConfiguration](https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-getconfiguration)
+  has been added to allow accessing the last configuration set by
+  [RTCPeerConnection.prototype.setConfiguration](https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-setconfiguration).
+* The `Opus` codec is now supported in mp4 (ISO-BMFF) containers with Media Source Extensions (MSE).
+* The new SourceBuffer.changeType() method allows for improved cross-codec or cross-bytestream transitions
+  within a SourceBuffer. It would for example allow changing the type of media bytes append to a media stream.
+
+## Network
+
+### Intervention Reports
+
+An "intervention" happens when the user agent (the browser) willfully violates
+the request of a web application in order to improve performance, security or
+eliminate behavior that is considered annoying to a user. These interventions
+can now be reported either by using the
```

Comment:
Would be better to be clear that it's about the web page reporting back to the page author. This is not a feature many web developers are familiar with.

Reply:
Added "back to the page author" after "reported".